Makefile

```diff
@@ -260,8 +262,8 @@ install-strip:
 	cd $(LIB_INSTALL_PATH) && $(STRIP_EXEC) $(DEP_TO_LIB)
 	cd $(LUA_INSTALL_CPATH) && $(STRIP_EXEC) *.so
 
-install: install-dep install-emulator install-strip $(INSTALL_PLAT) install-tests
-deb: install-dep install-emulator install-strip $(INSTALL_PLAT)
+install: install-dep install-emulator install-strip $(INSTALL_PLAT)
+deb: install
 	mkdir -p $(DESTDIR)/DEBIAN $(DESTDIR)/share/doc/cartesi-machine/
```

Comment:
The prefix is missing for statements in this target after the recent changes. Maybe use `SHARE_INSTALL_PATH` ?